src/AutoMapper/ProfileMap.cs

```diff
@@ -1,273 +1,275 @@
-                    .OfType<PrePostfixName>()
-                    .SelectMany(m => m.Prefixes)
-                    .ToArray();
-
-            Postfixes =
-                profile.MemberConfigurations
-                    .Select(m => m.NameMapper)
-                    .SelectMany(m => m.NamedMappers)
-                    .OfType<PrePostfixName>()
-                    .SelectMany(m => m.Postfixes)
-                    .ToArray();
-
-            _typeMapConfigs = profile.TypeMapConfigs.ToArray();
-            _openTypeMapConfigs = profile.OpenTypeMapConfigs.ToArray();
-        }
-
-
-        public bool AllowNullCollections { get; }
-        public bool AllowNullDestinationValues { get; }
-        public bool ConstructorMappingEnabled { get; }
-        public bool CreateMissingTypeMaps { get; }
-        public bool ValidateInlineMaps { get; }
-        public bool EnableNullPropagationForQueryMapping { get; }
-        public string Name { get; }
-        public Func<FieldInfo, bool> ShouldMapField { get; }
-        public Func<PropertyInfo, bool> ShouldMapProperty { get; }
-
-        public IEnumerable<Action<PropertyMap, IMemberConfigurationExpression>> AllPropertyMapActions { get; }
-        public IEnumerable<Action<TypeMap, IMappingExpression>> AllTypeMapActions { get; }
-        public IEnumerable<string> GlobalIgnores { get; }
-        public IEnumerable<IMemberConfiguration> MemberConfigurations { get; }
-        public IEnumerable<MethodInfo> SourceExtensionMethods { get; }
-        public IEnumerable<IConditionalObjectMapper> TypeConfigurations { get; }
-        public IEnumerable<string> Prefixes { get; }
-        public IEnumerable<string> Postfixes { get; }
-        public IEnumerable<ValueTransformerConfiguration> ValueTransformers { get; }
-
-        public TypeDetails CreateTypeDetails(Type type) => _typeDetails.GetOrAdd(type);
-
-        private TypeDetails TypeDetailsFactory(Type type) => new TypeDetails(type, this);
-
-        public void Register(IConfigurationProvider configurationProvider)
-        {
-            foreach (var config in _typeMapConfigs)
-            {
-                BuildTypeMap(configurationProvider, config);
-
-                if (config.ReverseTypeMap != null)
-                {
-                    BuildTypeMap(configurationProvider, config.ReverseTypeMap);
-                }
-            }
-        }
-
-        public void Configure(IConfigurationProvider configurationProvider)
-        {
-            foreach (var typeMapConfiguration in _typeMapConfigs)
-            {
-                Configure(typeMapConfiguration, configurationProvider);
-                if (typeMapConfiguration.ReverseTypeMap != null)
-                {
-                    Configure(typeMapConfiguration.ReverseTypeMap, configurationProvider);
-                }
-            }
-        }
-
-        private void BuildTypeMap(IConfigurationProvider configurationProvider, ITypeMapConfiguration config)
+using System;
```

Comment:
I know GitHub sometimes messes the diff, but if you could fix it somehow, that would be nice :)

Reply:
I'll see what I can do. I am assuming it's line endings, but no idea why it would affect only this file.

Reply:
I have force saved the file with CRLF line endings, which has produced a much smaller diff in GitHub.

I am guessing that this file has messed up line endings. Not sure what I can do beyond this. It's at least reviewable now, even if 90% of the changes are not really changes.

Reply:
👍